WordPress/openverse-frontend
# Add `new_header` flag

## Fixes


Fixes #1803 by @obulat

## Description


This PR adds a `new_header` feature flag which will contain all the work on the new header.

I renamed the current header directory and the component from `VHeader` to `VHeaderOld`, but did not rename the other components inside the new `VHeaderOld` directory (since they are different anyways because of the path, and the new components will probably have different names. If not, we can rename them later)

## Testing Instructions

Run `pnpm dev`. Everything should work as it currently does.
Go to `http://localhost:8443/preferences` and check the `new_header` flag:
<img width="545" alt="Screen Shot 2022-09-11 at 8 30 46 PM" src="https://user-images.githubusercontent.com/15233243/189541389-289a185b-b842-44d6-aa68-73ccc1d40ab3.png">

Now, open a search page. You should be seeing the new header, which currently is the same as the old one, except it doesn't have the pages menu and the Openverse wordmark:

<img width="770" alt="Screen Shot 2022-09-11 at 8 32 08 PM" src="https://user-images.githubusercontent.com/15233243/189541413-cecf96df-387e-4791-b447-1ff1f1fbec17.png">


## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
